UI/frontPage.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./assets/css/frontPage.css">
+  <link rel="shortcut icon" type="image/x-icon" href="./assets/images/logo.gif">
+  <title>politico : landing</title>
+</head>
+
+<body>
+
+        <nav class="politico-nav">
+                <div class="nav-container">
+                    <div class="nav-left">
+                        <a href="./frontPage.html"><h2>POLITICO</h2></a>
+                    </div>
+
+                    <ul class="header">
+                        <li class="login-btn">
+                            <a href="./signin.html">login</a> 
+                        </li>
+                        <li class="signup-btn"> 
+                           <a href="./signup.html">sign up</a> 
+                        </li>
+                    </ul>
+
+                </div>
+    
+            </nav>
+
+  <div class="main-wrapper home">
+    <div class="landing">
+      <h1> <span>Politico:  </span>   The online voting website</h1>
+      <a href="./Application.html">
+        <button class="btn btn-pri">Apply as a Candidate</button>
+      </a>
+    </div>
+  </div>
+  <div class="footer">
+    <p>&copy; 2019. Taiwo Oyindamola Ezekiel. All images were gotten from Google and are not mine.</p>
+  </div>
+
+</body>
+
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks